src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/IncidentType.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.incidentreporting.jpa
+
+enum class IncidentType {
```

Comment:
not asking for this to be changed, but i figure this too would need to be an entity in future…?

Reply:
Maybe?  But introducing a new incident type would require a lot of coordination from different teams, not sure it helps being a table as enum allow us to apply business rules or understand what the types are, anyway hard to know at this point!

Reply:
i was under the impression that the super-admins can modify report types in nomis… if we can get away with _not_ doing that YAY